jenkins/bc-wallet-demo/Jenkinsfile

```diff
@@ -32,20 +10,21 @@ String getImageTagHash(String imageName, String tag = "") {
 }
 
 node {
+  def config = load "../workspace@script/jenkins/config.groovy"
   stage("Building images ...") {
     script {
       openshift.withCluster() {
         openshift.withProject() {
 
           // Find all of the build configurations associated to the application using labels ...
-          def buildconfigs = openshift.selector("bc", [ app : "${APP_LABEL}" ])
-          echo "Found ${buildconfigs.count()} buildconfigs for app label (app=${APP_LABEL}): ${buildconfigs.names()}"
+          def buildconfigs = openshift.selector("bc", [ app : "${config.APP_LABEL}" ])
```

Comment:
`config.APP_LABEL` is not defined in `config.groovy`.

Reply:
I think I have it defined: https://github.com/wadeking98/BC-Wallet-Demo-Configurations/blob/31ac7235d64664c3f0ba27f3dcab1a246739dd39/jenkins/config.groovy#L13